docs/ReleaseNotes.md

```diff
@@ -1,50 +1,93 @@
 # IPDK Kubernetes* Infrastructure Offload Release Notes
 
-## What's New in This Release
+## Releases
 
-### IPDK 23.07 K8s-Infra-Offload Component
+### IPDK 23.07
 
-- This is the release of K8s-Infra-Offload recipe for ES2K and DPDK targets.
-This is the first release that supports both these targets.
+- This is the first release of K8s-Infra-Offload recipe that supports ES2K and DPDK targets.
 
 ### Highlights
 
-#### DPDK and ES2K targets
-- Support for Kubernetes Container Network Interface (CNI) to enable pods to
-  send/receive traffic.
-- Intra Node L3 Forwarding to enable pod to pod communication, on the same node,
-  via CNI interfaces.
+#### ES2K target
+
+- Support for Kubernetes Container Network Interface (CNI) to deploy pods and
+  enable pod-to-pod connectivity on a P4 target using hardware device interfaces.
+- Use of internal gateway with dummy MAC to enable layer-3 connectivity on the same node.
+- Support for SRIOV and dynamic Subfunctions on ES2K.
+  Subfunction is a lightweight function that has a parent PCI function on which it is
+  deployed. It is created and deployed in a unit of 1. Unlike SRIOV VFs, a subfunction
+  doesn't require its own PCI virtual function. A subfunction communicates with the
+  hardware through the parent PCI function.
+- Infra Manager build support on ARM cores.
```

Comment:
Do we really need to building Manager for ACC? Its not useful from deployment perspective with this release. And, we are not giving any steps on how to do this .... and I am not sure if this has been tested by validation team to claim such support.

Reply:
Yes. The dockerfile exists and has been tested on ACC by validation for rule config. It's not needed in current deployment but it's there.